cmd/populator/modules.go

```diff
@@ -19,13 +19,13 @@ end:
 
 func init() {
     modules = []Module{
-		{
-			name:        "liquibase-aws-extension",
-			category:    Extension,
-			owner:       "liquibase",
-			repo:        "liquibase-aws-extension",
-			artifactory: Maven{},
-		},
+	{
+	    name:        "liquibase-aws-extension",
+	    category:    Extension,
+	    owner:       "liquibase",
```

Comment:
These are not the correct settings for a Maven repository. Look at a Maven artifact and use that as the base. It won't work to just change Github to Maven. Maven doesn't have owners and repositories like GitHub